File: bin/Nodes/C/ReactionDiffusion.c
#include "Imogen.h"

typedef struct ReactionDiffusion_t
{
	float boost;
	float divisor;
	float colorStep;
	int PassCount;
	int size;
} ReactionDiffusion;

int main(ReactionDiffusion *param, Evaluation *evaluation, void *context)
{
    if (!(evaluation->dirtyFlag & DirtyParameter))
    {
		return EVAL_OK;
    }
    
	SetEvaluationSize(context, evaluation->targetIndex, 256<<param->size, 256<<param->size);
	return EVAL_OK;
}

File: bin/Nodes/JS/ReactionDiffusion.js
function ReactionDiffusion(parameters, evaluation, context)
{
    if (!(evaluation.dirtyFlag & DirtyParameter))
    {
		return EVAL_OK;
    }
    
	SetEvaluationSize(context, evaluation.targetIndex, 256<<parameters.size, 256<<parameters.size);
	return EVAL_OK;
}